src/pages/explorePage/ExplorePage.tsx

```diff
@@ -99,6 +138,41 @@ const ExplorePageContent = ({
     hasPrereqs,
   };
 
+  const mapFilterStateToURL = (fs: SearchFilterState) => {
+    return {
+      exclude: fs.courseCodes
```

Comment:
Is it easy to make this an array in the URL so it's not as verbose/long?

Reply:
Definitely. Changed to reflect array!